src/main/java/net/fabricmc/loom/configuration/providers/forge/FieldMigratedMappingConfiguration.java

```diff
@@ -139,39 +130,46 @@ public void updateFieldMigration(Project project) throws IOException {
 			});
 			Files.writeString(migratedFieldsCache, new Gson().toJson(map));
 			Files.deleteIfExists(tinyMappings);
+			Files.deleteIfExists(tinyMappingsWithSrg);
 		}
 
-		if (!Files.exists(tinyMappings)) {
+		if (Files.notExists(tinyMappings) || Files.notExists(tinyMappingsWithSrg)) {
 			Table<String, String, String> fieldDescriptorMap = HashBasedTable.create();
 
 			for (Map.Entry<FieldMember, String> entry : migratedFields) {
 				fieldDescriptorMap.put(entry.getKey().owner, entry.getKey().field, entry.getValue());
 			}
 
-			MemoryMappingTree mappings = new MemoryMappingTree();
+			injectMigration(project, fieldDescriptorMap, rawTinyMappings, tinyMappings);
+			injectMigration(project, fieldDescriptorMap, rawTinyMappingsWithSrg, tinyMappingsWithSrg);
```

Comment:
Thanks for this, it makes applying the same thing to the Mojang-merged mappings much easier too